src/app/modules/form/admin-form/admin-form.utils.ts

```diff
@@ -501,9 +501,10 @@ export const verifyValidUnicodeString = (value: any, helpers: any) => {
  */
 export const verifyUserBetaflag = (
   user: IUserSchema,
+  globalUserBeta: boolean,
   betaFlag: keyof Exclude<IUserSchema['betaFlags'], undefined>,
 ) => {
-  return user?.betaFlags?.[betaFlag]
+  return user?.betaFlags?.[betaFlag] || globalUserBeta
```

Comment:
Thinking about this again, I think the naming is wrong. TYhe global beta flag actually activates the feature for the whole platform, to it's really a feature flag, not a beta flag. 

Beta does work for a group of users or agencies, so having beta flags in user objects, or in agency objects works, but global beta sounds weird 🤔 

Just for terminology completeness, the way the global flags are implemented now is indeed not a kill switch, because turning them off still lets the beta user use it.

(Not Related to PR: I'm thinking we should actually discuss kill switches for FormSG at some point. With feature like payment, it makes sense to me to be able to turn all payment off if something catastrophic happens 🤔 )

Reply:
@timotheeg so for this, should all the related names be "global feature flag" instead of "global beta flag"?

Reply:
Yes Mam'! 😃

Reply:
In fact, "feature flag" is probably enough

Reply:
Done in [b2dd2f3](https://github.com/opengovsg/FormSG/pull/6286/commits/b2dd2f3b5175d5c82b91039d2fbbf0928eb7840d), [90362ee](https://github.com/opengovsg/FormSG/pull/6286/commits/90362eee30347c1582b1bdb49bb73e886a2a4a91) and [78f8b6b](https://github.com/opengovsg/FormSG/pull/6286/commits/78f8b6bfe2f5f7710152aa3014fc1a619ae9fa5e).